app/views/unattended/provisioning_templates/snippet/preseed_networking_setup.erb

```diff
@@ -30,7 +29,7 @@ iface $real inet <%= host_dhcp ? 'dhcp' : 'static' %>
     dns-nameservers <%= host_subnet.dns_servers.join(' ') %>
     dns-search <%= @host.domain %>
 <% end -%>
-<% if @host.ip6 && host_subnet6 && !host_dhcp6 -%>
+<% if @host.ip6 && host_subnet6&.dhcp_boot_mode? -%>
```

Comment:
I think this is wrong (because the next line says it's static), but then I noticed more of the preseed templates are IMHO wrong so I just stuck to using the safe navigator.

Reply:
I think those are not equivalent?
before, we had:
```ruby
host_dhcp6 = host_subnet6.nil? ? true : host_subnet6.dhcp_boot_mode?
if @host.ip6 && host_subnet6 && !host_dhcp6 
…
```
Which meant: if the host has v6 address and host has v6 subnet and not `host_subnet6.dhcp_boot_mode?`
Now it reads: if the host has v6 address and has v6 subnet and the `host_subnet6.dhcp_boot_mode?`

So you lost a "not" here?

```suggestion
<% if @host.ip6 && !host_subnet6&.dhcp_boot_mode? -%>
```

Reply:
Ah, that explains why I thought it was wrong.